lib/index.ts

```diff
@@ -522,6 +512,15 @@ export default class Backburner {
     this._ensureInstance();
   }
 
+  private handleError(e) {
```

Comment:
I don't see the benefit of the method here since it is only used once...

Reply:
made it inline